pkg/apis/networking/types.go

```diff
@@ -60,10 +71,18 @@ type NetworkPolicySpec struct {
 	// allowed if there are no NetworkPolicies selecting the pod (and cluster policy
```

Comment:
'...if there are no NetworkPolicies with "Egress" PolicyType selecting the pod...'. Likewise for Ingress doc